.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+    - 2.7
+install:
+    - "pip install tox"
+env:
+    - TOX_ENV=django18
```

Comment:
is it capture the django versions in between? django18-django111

Reply:
I left out 9 and 10 because figured it wasn't necessary. If there's a change in one of those two it would probably appear in 11 as well